modules/admin_manual/pages/configuration/user/oidc/index.adoc

```diff
@@ -3,56 +3,211 @@
 
-== Setup Service Discovery
+The above configuration assumes that the OpenId Provider is supporting xef:setup-service-discovery[service discovery].
+If the OpenId Provider _does not_ support service discovery, the endpoint configuration has to be done manually, as in the following example.
+
+[source,php]
+----
+include::{examplesdir}configuration/user/oidc/config-without-service-discovery.php[]
+----
+
+TIP: Please refer to xref:configuration/server/config_sample_php_parameters.adoc#oidc-configuration[the OIDC configuration example] in the sample configuration documentation for more information on how to configure ownCloud to work with OIDC.
+
+=== Setup Within the OpenID Provider
+
+When registering ownCloud as an OpenID client, use the following URLs:
+
+[cols="2"]
+|===
+|**The redirect URL**
+|`https://cloud.example.net/index.php/apps/openidconnect/redirect`
+|**The logout URL** +
+//(within the client registration of the OpenID Provider).
+Only use this if {openid-connect-frontchannel-logout-url}[OpenID Connect Front-Channel Logout 1.0] is supported.
+|`https://cloud.example.net/index.php/apps/openidconnect/logout` 
+|===
+
+=== Setup Service Discovery
 
 To allow other clients to use OpenID Connect when talking to ownCloud, please set up a redirect on the webserver to point `.well-known/openid-configuration` to `/index.php/apps/openidconnect/config`.
-Here is an .htaccess example
+Here is an example of how to do so in .htaccess.
 
 [source]
 ----
 RewriteRule ^\.well-known/openid-configuration /index.php/apps/openidconnect/config [R=301,L]
 ----
 
-//== Integration with different IdPs
-// (e.g., Ping Identity / Kopano Konnect / Keycloak)
+TIP: Please refer to https://httpd.apache.org/docs/current/mod/mod_rewrite.html[the mod_rewrite manual] for further details on creating rewrite rules.
+
+=== Integrating with Different Identity Providers (IdPs)
+
+There are a number of different IdPs available; some open source, and some commercial.
+Below, you can find sample configurations for working with three of them: xref:kopano-konnect[Kopano Konnect], xref:keycloak[Keycloak], and xref:ping-identity[Ping Identity].
+
+==== Kopano Konnect
+
+[source,php]
+----
+"system": {
+    "openid-connect": {
+        "provider-url": "https://$KOPANO_KONNECT_DOMAIN",
+        "client-id": "ownCloud",
+        "client-secret": "ownCloud",
+        "loginButtonName": "Kopano",
+        "autoRedirectOnLoginPage": false,
+        "redirect-url": "https://$OWNCLOUD_DOMAIN/index.php/apps/openidconnect/redirect",
+        "mode": "userid",
+        "search-attribute": "preferred_username",
+        "use-token-introspection-endpoint": false
+    },
+    "debug": true
+}
+----
+
+TIP: If you're interested, the https://github.com/owncloud-docker/compose-playground/tree/master/kopano/konnect[ownCloud Composer Playground repository] shows how to setup Kopano Konnect and configure it with the Caddy webserver and ownCloud. 
```

Comment:
Better use of {...link...} instead http://

Reply:
Well spotted. Thanks.